core/common/src/main/java/alluxio/grpc/GrpcChannelBuilder.java

```diff
@@ -236,6 +237,11 @@ public GrpcChannel build() throws AlluxioStatusException {
       // Release the managed channel to the pool before throwing.
       GrpcManagedChannelPool.INSTANCE().releaseManagedChannel(mChannelKey,
           mConfiguration.getMs(PropertyKey.NETWORK_CONNECTION_SHUTDOWN_TIMEOUT));
+      if (exc instanceof UnavailableException) {
+        // Override exception from authentication layer.
+        throw new UnavailableException(
+            String.format("Target Unavailable. %s", mChannelKey.toStringShort()));
```

Comment:
Is there any info we want to retain from the original exception (ie stack trace?)